akka-persistence/src/main/scala/akka/persistence/fsm/PersistentFSM.scala

```diff
@@ -54,6 +54,16 @@ trait PersistentFSM[S <: FSMState, D, E] extends PersistentActor with Persistent
   private var currentStateTimeout: Option[FiniteDuration] = None
 
   /**
+   * "akka.persistence.fsm.enable-snapshot-after"
+   */
+  private val enableSnapshotAfter = context.system.settings.config.getBoolean("akka.persistence.fsm.enable-snapshot-after")
+
+  /**
+   * "akka.persistence.fsm.snapshot-after"
+   */
+  private val snapshotAfter = context.system.settings.config.getInt("akka.persistence.fsm.snapshot-after")
```

Comment:
This is the same type of binary compatibility concern/question as in https://github.com/akka/akka/pull/22615#discussion_r112696828